client/src/components/requester_order/requesterOrderCtrl.js

```diff
@@ -130,8 +130,6 @@
       ajaxFactory.getDirections(currentLocation, meetingLocation)
         .then(function(response) {
 
-          //add marker for source
-          L.marker([currentLocation[0], currentLocation[1]], { icon: pinIcon} ).addTo(vm.map);
           //add marker for destination
           L.marker([meetingLocation[0], meetingLocation[1]], { icon: pinIcon} ).addTo(vm.map);

```

Comment:
I know it's not part of your commit, but can you add the space after `pinIcon`? thanks